test.yml

```diff
@@ -1,6 +1,7 @@
 ---
 - hosts: localhost
   remote_user: root
```

Comment:
@leseb thanks, I'll try that. I think I'd prefer to just remove it though so it uses the current user.